integrations/uphold/src/main/java/org/dash/wallet/integration/uphold/ui/UpholdSplashActivity.java

```diff
@@ -45,6 +45,7 @@
 import kotlin.Unit;
 
 @AndroidEntryPoint
+// TODO: move this into IntegrationOverviewFragment
```

Comment:
I wanted to use the same fragment for both Coinbase and Uphold overview since they use the same layout, however, since UpholdSplashActivity is called by a callback to finish authentication it's hard to do it this way right now - needs some more refactoring. Leaving it be for now.